tests/lean/hashmap_on_disk/lakefile.lean

```diff
@@ -0,0 +1,22 @@
+import Lake
```

Comment:
I guess we should generate the `lakefile.lean` files

Reply:
I can do it, but I'm not sure what we should put in the various fields

Reply:
basically everything I have so far is (I believe) useful -- just replace the occurrences of `hashmap` with `<name-of-the-current-file>`

you probably want to keep the mathlib import, I currently need it for `run_cmd`, it's a little heavy-handed, but convenient

Reply:
I automated the generation of the lakefile.lean files.
For now:
- the package name is the crate name in snake case
- there is no default target